[2024-11-30T05:27:46.851+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-30T05:27:46.883+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:27:18.121775+00:00 [queued]>
[2024-11-30T05:27:46.891+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:27:18.121775+00:00 [queued]>
[2024-11-30T05:27:46.892+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-30T05:27:46.906+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2024-11-30 05:27:18.121775+00:00
[2024-11-30T05:27:46.915+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt-dag', 'dbt_snapshot', 'manual__2024-11-30T05:27:18.121775+00:00', '--job-id', '329', '--raw', '--subdir', 'DAGS_FOLDER/dbt-dag.py', '--cfg-path', '/tmp/tmp4aq5nd9e']
[2024-11-30T05:27:46.917+0000] {standard_task_runner.py:91} INFO - Job 329: Subtask dbt_snapshot
[2024-11-30T05:27:46.939+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4235) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-30T05:27:46.940+0000] {standard_task_runner.py:63} INFO - Started process 4236 to run task
[2024-11-30T05:27:46.968+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:27:18.121775+00:00 [running]> on host b09a94790db6
[2024-11-30T05:27:47.103+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt-dag' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2024-11-30T05:27:18.121775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-30T05:27:18.121775+00:00'
[2024-11-30T05:27:47.104+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-30T05:27:47.118+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-30T05:27:47.121+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2024-11-30T05:27:47.132+0000] {subprocess.py:86} INFO - Output:
[2024-11-30T05:27:49.144+0000] {subprocess.py:93} INFO - [0m05:27:49  Running with dbt=1.8.7
[2024-11-30T05:27:50.202+0000] {subprocess.py:93} INFO - [0m05:27:50  Registered adapter: snowflake=1.8.4
[2024-11-30T05:27:50.788+0000] {subprocess.py:93} INFO - [0m05:27:50  Found 7 models, 1 test, 4 sources, 459 macros
[2024-11-30T05:27:50.792+0000] {subprocess.py:93} INFO - [0m05:27:50
[2024-11-30T05:27:50.793+0000] {subprocess.py:93} INFO - [0m05:27:50  Nothing to do. Try checking your model configs and model specification args
[2024-11-30T05:27:51.579+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-30T05:27:51.581+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-30T05:27:51.608+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt-dag, task_id=dbt_snapshot, run_id=manual__2024-11-30T05:27:18.121775+00:00, execution_date=20241130T052718, start_date=20241130T052746, end_date=20241130T052751
[2024-11-30T05:27:51.650+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-30T05:27:51.671+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-30T05:27:51.674+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
